src/common/docker-compose/index.ts

```diff
@@ -116,6 +117,8 @@ export class DockerComposeUtils {
           '--providers.docker.exposedByDefault=false',
           `--providers.docker.constraints=Label(\`traefik.port\`,\`${gateway_port}\`)`,
           ...(use_ssl ? [
+            // Ignore local certs being invalid on proxy
+            `--serversTransport.insecureSkipVerify=true`,
```

Comment:
I feel like we should allow for https protocols and use a tcp router (ssl passthrough), but maybe this is the stop gap solution?

Reply:
So I was thinking about this, but by not using the TCP router we actually still get a valid certificate locally for the first hop.

So client <-> traefik <-> service.
While the `traefik <-> service` may not be valid ssl, the `client <-> traefik` is going to be valid ssl since we terminate the ssl with our valid cert and then just passthrough.

Reply:
Will the connection still be secure? Sounds like it, but I wanted to confirm to make sure secure cookies for example still work.

Reply:
Everything appears to work fine. Secure cookies work exactly as expected and the client and server both don't complain.